pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -160,7 +157,32 @@ func (r *ReconcileApplicationMonitoring) Reconcile(request reconcile.Request) (r
 
 func (r *ReconcileApplicationMonitoring) reconcileConfig(cr *applicationmonitoringv1alpha1.ApplicationMonitoring) (reconcile.Result, error) {
 	log.Info("Phase: Reconciling Config")
-	return reconcile.Result{}, r.createOrUpdateAdditionalScrapeConfig(cr)
+	err := r.syncBlackboxTargets(cr)
+	return reconcile.Result{RequeueAfter: time.Second * ReconcilePauseSeconds}, err
+}
+
+func (r *ReconcileApplicationMonitoring) syncBlackboxTargets(cr *applicationmonitoringv1alpha1.ApplicationMonitoring) error {
+	log.Info("Phase: Reconciling Config syncBlackboxTargets")
+
+	previous := cr.Status.Bbthash
+
+	bbtList := common.GetBTConfig()
```

Comment:
The variable names are a bit cryptic, maybe use longer names or try to avoid the variables and just use e.g. `common.GetBTConfig` inline.